content/en/docs/setup/getting-started/index.md

```diff
@@ -327,11 +327,10 @@ $ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -
 
 ### Verify external access {#confirm}
 
-Confirm that the Bookinfo application is accessible from outside. Copy the
-output of this command and paste into your browser:
+Confirm that the Bookinfo application is accessible from outside.
```

Comment:
```suggestion
Confirm that the Bookinfo application is accessible from outside
by viewing the Bookinfo product page using a browser.

1. Run the following command to retrieve the external address of the Bookinfo application.

    {{< text bash >}}
    $ echo http://$GATEWAY_URL/productpage
    {{< /text >}}
    
1. Paste the output from the previous command into your web browser and confirm that the Bookinfo product page is displayed.
```

Reply:
Looks good, Also adding the output of the `echo` to describe how the output will like.

`http://192.168.99.100:32194/productpage`

Reply:
Thanks @frankbu